### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by bashu**
are you sure it's not a typo?

### CODE REVIEW

**Comment by atodorov** [comment]
@jacobian this has been open for a long time and looks good to me.

Additionally I see a few more open PRs and it looks like they fell behind the sofa. I volunteer to maintain this package if you folks are busy. Can send you an email or open a separate issue about this though.

File: django_comments/views/comments.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def post_comment(request, next=None, using=None):
                 escape(ctype), escape(object_pk)))
     except (ValueError, ValidationError) as e:
         return CommentPostBadRequest(
-            "Attempting go get content-type %r and object PK %r exists raised %s" % (
+            "Attempting go get content-type %r and object PK %r raised %s" % (
```
Comment (ObserverOfTime) [inline django_comments/views/comments.py:69]:
```suggestion
            "Attempting to get content-type %r and object PK %r raised %s" % (
```

Comment (claudep) [reply django_comments/views/comments.py:69]: Ahhhh... now I understand :-) Thanks!

**Comment by claudep** [comment]
Thanks, pushed in b0a7a9e

**Pull request closed without merging**